util/lockwaiter/lockwaiter.go

```diff
@@ -189,7 +189,6 @@ func (lw *Manager) WakeUp(txn, commitTS uint64, keyHashes []uint64) {
 	// wake up delay waiters, this will not remove waiter from queue
 	if len(wakeUpDelayWaiters) > 0 {
 		for _, w := range wakeUpDelayWaiters {
-			w.LockTS = txn
```

Comment:
Why it is added in the first place?

Reply:
I don't remember the reason at that time..